File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        for head in range(len(nums)):
            for head1 in range(head+1, len(nums)):
                if nums[head] > nums[head1]:
                    nums[head], nums[head1] = nums[head1], nums[head]
                

File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int start = 0;
        int curr = 0;
        int end = nums.size() - 1;
        while (curr <= end){
            if (nums[curr] == 0){
                swap(nums[curr],nums[start]);
                curr++;
                start++;
            }
            else if (nums[curr] == 1){
                curr++;

            }
            else{
                swap(nums[curr],nums[end]);
                end--;
                
            }

            }
        }
    };